src/pages/steps/EventTypeAndThemeStep.tsx

```diff
@@ -230,6 +250,11 @@ const EventTypeAndThemeStep = ({
     [getTypesByScopeQuery.data, sortByLocalizedName],
   );
 
+  const cultuurkuurTypes = useMemo(
+    () => types.filter((type) => CULTUURKUUR_TYPE_IDS.includes(type.id)),
+    [types],
+  );
```

Comment:
Since we already memoize the building of the `types` I would maybe merge the two. Basically move the ternary in the JSX to the original useMemo and have it memoize the correct types on the first go